src/main/java/org/gridsuite/shortcircuit/server/utils/FaultResultSpecificationBuilder.java

```diff
@@ -28,8 +32,27 @@ public static Specification<FaultResultEntity> resultUuidEquals(UUID value) {
     }
 
     public static Specification<FaultResultEntity> buildSpecification(UUID resultUuid, List<ResourceFilter> resourceFilters) {
-        Specification<FaultResultEntity> specification = Specification.where(resultUuidEquals(resultUuid));
-        return SpecificationUtils.appendFiltersToSpecification(specification, resourceFilters);
+        return (final Root<FaultResultEntity> root, final CriteriaQuery<?> query, final CriteriaBuilder builder) -> {
+            query.distinct(true);
+
+            Specification<FaultResultEntity> specification = Specification.where(resultUuidEquals(resultUuid));
+            List<ResourceFilter> parentsFilters = resourceFilters.stream()
+                    .map(filter -> {
+                        if (FeederResultEntity.Fields.connectableId.equals(filter.column())) {
+                            // this column belong to the feederResults "sub-columns"
+                            // as a resource filter they will filter out when a FaultResult has no feeder result matching the filter
+                            // BUT individual feeders are not filtered within each Fault Result
+                            return new ResourceFilter(filter.dataType(),
+                                    filter.type(),
+                                    filter.value(),
+                                    "feederResults." + filter.column());
```

Comment:
I would use `FeederResultEntity.Fields` for both

Reply:
I think we should do just like the other services, let the frontend pass "feederResults." prefix instead

Reply:
Agreed. Moved all that prefix logic to the frontend (which already handles column ids association anyway).